experimental/origin-dapp2/package.json

```diff
@@ -16,8 +16,9 @@
     "build:js": "NODE_ENV=production webpack --progress",
     "build:css": "node scripts/getCss > public/app.css",
     "build": "npm run build:css && npm run build:js",
-    "lint": "eslint .",
+    "lint": "eslint . && npm run prettier:check",
     "prettier": "find ./src -iname '*.js' | xargs ./node_modules/.bin/prettier --write",
+    "prettier:check": "find ./src -iname '*.js' | xargs ./node_modules/.bin/prettier -c",
```

Comment:
Is there any reason to do it this way instead of `prettier -c **/*.js`?

Reply:
I've definitely been bit in JS projects with shell expansions that can't hold all the matching .js files.

Reply:
From a shell it needs to be `"**/*.js"` to prevent shell expansion, but if it is in `package.json` it shouldn't matter

Reply:
Updated this 🙂